index.bs

```diff
@@ -9939,19 +9939,14 @@ allowed.  The security check takes the following four inputs:
 
```

Comment:
Github sucks and won't let me comment on the line I want to comment on, but the security checks now takes _three_ inputs, not four.